packages/backend/src/graphql/resolvers/account.test.ts

```diff
@@ -0,0 +1,81 @@
+import { gql } from 'apollo-server-koa'
+import Knex from 'knex'
+
+import { createTestApp, TestContainer } from '../../tests/app'
+import { IocContract } from '@adonisjs/fold'
+import { AppServices } from '../../app'
+import { initIocContainer } from '../..'
+import { Config } from '../../config/app'
+import { truncateTables } from '../../tests/tableManager'
+import { AccountService } from '../../account/service'
+import { Account as AccountModel } from '../../account/model'
+import { Account } from '../generated/graphql'
+
+describe('Account Resolvers', (): void => {
+  let deps: IocContract<AppServices>
+  let appContainer: TestContainer
+  let knex: Knex
+
+  beforeAll(
+    async (): Promise<void> => {
+      deps = await initIocContainer(Config)
+      appContainer = await createTestApp(deps)
+      knex = await deps.use('knex')
+    }
+  )
+
+  afterAll(
+    async (): Promise<void> => {
+      await appContainer.apolloClient.stop()
+      await appContainer.shutdown()
+      await truncateTables(knex)
```

Comment:
`knex.destroy()` is called in `appContainer.shutdown()`.
Does that mean `truncateTables(knex)` should happen first?
Ditto for other (some pre-existing) test files.

Reply:
Thanks I changed the ordering to ensure truncate should happen first